### ISSUE TITLE
metadata-generation-failed (when installing using pip inside linux)

### ISSUE THREAD

**Issue by jonardcaguioa** [ultrafunkamsterdam/undetected-chromedriver#514]
Collecting undetected-chromedriver
  Using cached undetected-chromedriver-3.1.3.tar.gz (39 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      ERROR: Can not execute `setup.py` since setuptools is not available in the build environment.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

**Comment by sebdelsol** [comment]
Are you sure installing an old version is the solution ?

**Comment by Eagle19243** [comment]
Yes, I got the same error and the error was gone when I installed `3.0.3` or `3.1.0`. Not the latest version.
